spirv_msl.cpp

```diff
@@ -9688,6 +9694,31 @@ void CompilerMSL::fix_up_shader_inputs_outputs()
 	});
 }
 
+// Returns the number of IDs that Metal would assign to an inline uniform block in an argument buffer.
+uint32_t CompilerMSL::get_inline_uniform_block_binding_stride(SPIRType &type)
```

Comment:
Would this be necessary if we make sure to place the uniform buffer block at the highest ID? Then stride becomes irrelevant. I think relying on this layout is a little too sketchy for my liking.

Reply:
I imagine this isn't a commonly used feature. Really, I'm only adding it because support for this extension was added to MoltenVK, so now I feel like I have to support it with argument buffers. I guess this would work OK.

Reply:
Alright, I think the missing piece here is to move the (single) inline uniform block to the back of the argument buffer, and make sure it works for automatically allocated IDs as well as the add_msl_resource API.

Reply:
With that change I think this method is redundant.

Reply:
Done.